cockroachcloud/security-overview.md

```diff
@@ -13,7 +13,7 @@ The following table summarizes the CockroachCloud security features and provides
 Security feature | Description
 -------------|------------
 [Authentication](authentication.html) | <ul><li>Inter-node and node identity authentication using TLS 1.2</li><li>Client identity authentication using TLS 1.2 or username/password</li></ul>
-Encryption | <ul><li>Encryption in flight using TLS 1.2</li><li>The backups for AWS clusters are encrypted using [AWS S3’s server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) and the backups for GCP clusters are encrypted using [Google-managed server-side encryption keys](https://cloud.google.com/storage/docs/encryption/default-keys) </li></ul>
+Encryption | <ul><li>Encryption-in-flight using TLS 1.2</li><li>The backups for AWS clusters are encrypted using [AWS S3’s server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) and the backups for GCP clusters are encrypted using [Google-managed server-side encryption keys](https://cloud.google.com/storage/docs/encryption/default-keys) </li><li>All data on CockroachCloud is encrypted-at-rest using the tools provided by the cloud provider that your cluster is running in (i.e., [persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption) for GCP and [EBS encryption-at-rest](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html) for AWS). However, we do not enable encryption-at-rest within CockroachDB by default, so encryption will appear to be disabled in the [DB Console](../stable/ui-overview.html) since it is unaware of cloud provider encryption.</li></ul>
```

Comment:
Same comment as above re: finding it a bit more explicit to say something like "CockroachDB's internal implementation of e-a-r"

I might revise this

> The backups for AWS clusters are encrypted using AWS S3’s server-side encryption and the backups for GCP clusters are encrypted using Google-managed server-side encryption keys

to use bullets for scannability and be more compare/contrast-y with encryption-in-flight to explicitly mention encryption-at-rest so it shows up if I do a 'Ctrl-F', something like

> - The backups for AWS clusters are encrypted-at-rest using AWS S3’s server-side encryption
> - The backups for GCP clusters are encrypted-at-rest using Google-managed server-side encryption keys